src/main/resources/application.properties

```diff
@@ -34,4 +34,6 @@ design.runner=${design.runner}
 design.runner.license.service=${design.runner.license.service}
 
 spring.jackson.mapper.default-view-inclusion=true
-bms.version=${project.version}
\ No newline at end of file
+bms.version=${project.version}
+pagedresult.max.page.size=${pagedresult.max.page.size}
+pagedresult.default.page.size=${pagedresult.default.page.size}
```

Comment:
Should we remove this:
`pagedresult.default.page.size=${pagedresult.default.page.size} `

Reply:
Will remove not used property (Defaultpagesize)